endoreg_db/views/anonymization/validate.py

```
@@ -0,0 +1,66 @@
+    @transaction.atomic
+    def post(self, request, file_id: int):
+        # Serializer-Validierung mit deutscher Datums-Priorität
+        serializer = SensitiveMetaValidateSerializer(data=request.data or {})
+        serializer.is_valid(raise_exception=True)
+        payload = serializer.validated_data
+        payload.setdefault("is_verified", True)
+
+        # Try Video first
+        if payload.get("file_type") == "video" or not payload.get("file_type"):
+            video = VideoFile.objects.filter(pk=file_id).first()
+        if video:
```

Pylint (4.0.0)

[error] Possibly using variable 'video' before assignment

(E0606)